internal/gengapic/testdata/foo_rest_client_init.want

```diff
@@ -72,51 +72,7 @@ func (c *FooClient) TestIamPermissions(ctx context.Context, req *iampb.TestIamPe
 
 // Methods, except Close, may be called concurrently. However, fields must not be modified concurrently with method calls.
 type fooRESTClient struct {
-	host string
+	endpoint string
```

Comment:
So where are the generated methods like `Zip()` now?

Reply:
They were being generated incorrectly in the test. It's a longish story:
* Turns out showcase generation didn't turn on rest client surface
* Turning on rest transport for showcase revealed that method generation was erroneously happening twice
* Resolving the issue in favor of the grpc-like implementation means that the golden files no longer contain concrete transport methods
* Generating rest transport surface for showcase revealed other bugs